### DISCUSSION TITLE
how exactly is the Libre 3 calibrated?

### DISCUSSION THREAD

**Discussion by xialff** [maheini/FreeStyle-Libre-3-patch#44] [Q&A]
I see this phrase "The sensor can be calibrated in the range of -40 mg/dl to +20 mg/dl (-2,2 mmol/l to +1,1 mmol/l) to adjust differences between finger prick measurements and sensor readings." in the instructions, how exactly is the Libre 3 calibrated?

**Comment by maheini** [comment]
The libre 3 never has been calibrated in the instuctions, however, there are many ways:

- Via xDrip+: If connected to Juggluco, xDrip+ can be used to calibrate the readings
- Via Juggluco: As stated in your Juggluco response, there are safety restrictions whilst calibration of the sensor - there shouldn't be a to big gap between calibration and sensor.

Hope  this helps. If you got any questions, feel free to ask

**Comment by ga-zelle** [comment]
I was not aware of this repository. Interestingly I am prototyping Libre calibration directly in AAPS so I get along without xDrip. Similar to what I read in xDrip I use an offset and a slope parameter.
